src/main/java/org/springframework/data/support/IsNewStrategyFactorySupport.java

```diff
@@ -55,5 +56,6 @@ public final IsNewStrategy getIsNewStrategy(Class<?> type) {
 	 * @param type will never be {@literal null}.
 	 * @return the {@link IsNewStrategy} to be used for the given type or {@literal null} in case none can be resolved.
 	 */
+	@Nullable
 	protected abstract IsNewStrategy doGetIsNewStrategy(Class<?> type);
```

Comment:
The method is used internally only anyway as we throw an exception in case of `null` being returned from the template method. I wanted to avoid introducing a null object for a type which — through all public API — will never be null in the first place.